_newsletters/2023-04-01-april-2023.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+category: newsletters
+title: April 2023 Newsletter
+filename-custom: april-2023
+last_modified_at: May 1, 2023
+sidenav: about_us
+newsletter-year: 2023
+description: "Search team update, customer satisfaction, Google Analytics 4, release notes."
+---
+## Search team update
+
+### Welcome Jim Kreft, our new Search Experience Designer
+
+We’re very excited to announce that Jim Kreft has joined our team as Search Experience Designer. Jim is a data curation librarian with deep experience in building usable, sustainable cross agency data integration tools. He will be heading up our indexing and search experience design work, supporting technical questions and requirements from our customers. He’s excited to help make the Search.gov experience more valuable to our agency partners and to their users. Welcome Jim!
+
+**Product management**
+Amani Farooque, our contract Product Manager, left Search.gov this week for a new position. Many of you interacted with her on technical questions to our helpdesk and in meetings. We look forward to introducing you to our new contract Product Manager Iya Iduozee in the near future.
+
+## Customer Satisfaction
+### Share your feedback with us!
+
+**Let us know about your searchers’ satisfaction**
+Your website visitors’ search experience is very important to us. Please feel free to share any feedback you collect from your searchers. We will gladly discuss any concerns or questions you may have about this information. You can share your feedback by sending it to [search@gsa.gov](search@gsa.gov), or let us know if you would like to schedule a call.
+
+**Search.gov Annual Survey**
+Every year our parent organization, GSA’s Federal Acquisition Service, conducts a customer survey. This is one of the main ways we can learn your feedback, and let our leadership know what you think. You should soon be receiving an email with the subject line “How are we doing? Let us know!” from GSA’s address [surveys@research.gsa.gov] (surveys@research.gsa.gov). If you don’t see it, we would appreciate it if you could search in your email to locate it.
```

Comment:
Please add a space between the header and this line. like 

```
**Search.gov Annual Survey**

Every year etc.
```

Reply:
Also, please remove the space in the middle of `[surveys@research.gsa.gov] (surveys@research.gsa.gov)`, so it looks like `[surveys@research.gsa.gov](surveys@research.gsa.gov)`